### PR TITLE
PR: Reenable path manager sync on windows

### LINKED ISSUE #10686: Fix path manager synchronize on windows
After https://github.com/spyder-ide/spyder/pull/10623/, the sync was disabled. Need to reenable and make sure tests are correctly passing.

### PR DESCRIPTION
**Pull request by goanpeca**
## Description of Changes

* [x] Wrote at least one-line docstrings (for any new functions)
* [x] Added unit test(s) covering the changes (if testable)

* [x] Included a screenshot or animation (if affecting the UI, see [Licecap](https://www.cockos.com/licecap/))

### Issue(s) Resolved

Fixes #10686

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct: @goanpeca

### CODE REVIEW

**Comment by goanpeca** [comment]
@jnsebgosselin could you check it is working as expected?

Thanks!

File: spyder/app/mainwindow.py [hunk 1]
```diff
@@ -3013,7 +3013,7 @@ def show_path_manager(self):
 
         # TODO: update sync on windows
```
Comment (ccordoba12) [inline spyder/app/mainwindow.py]: Please remove this comment.

**Comment by ccordoba12** [comment]
@jnsebgosselin, is this working for you? Please let us know so we can merge this one.

**Comment by jnsebgosselin** [comment]
> 
> 
> @jnsebgosselin, is this working for you? Please let us know so we can merge this one.

I have tested it locally and everything seems to work as expected. Thank you very much!

**Comment by goanpeca** [comment]
This is ready @ccordoba12 !

**Pull request merged**